code/site/components/com_pages/view/html.php

```diff
@@ -143,15 +143,22 @@ public function getTitle()
     public function getMetadata()
     {
         $metadata = array();
-        if($data = $this->getPage())
+        if($page = $this->getPage())
         {
-            if(isset($data->metadata)) {
-                $metadata = KObjectConfig::unbox($data->metadata);
-            }
+            if($page->metadata)
+            {
+                $metadata = KObjectConfig::unbox($page->metadata);
 
-            //Set the description into the metadata if it doesn't exist.
-            if(!empty($data->summary) && !isset($data->metadata->description)) {
-                $metadata['description'] = $data->summary;
+                if($page->metadata->has('og:type'))
+                {
```

Comment:
@johanjanssens The older version sets the meta description too. It seems to be removed here. Intentional?

Reply:
@johanjanssens never mind